cocos/platform/ios/CCDevice-ios.mm

```diff
@@ -246,24 +246,34 @@ - (void)accelerometer:(CMAccelerometerData *)accelerometerData
     _acceleration->timestamp = accelerometerData.timestamp;
 
     double tmp = _acceleration->x;
-
-    switch ([[[UIApplication sharedApplication].windows[0] windowScene] interfaceOrientation])
+    UIInterfaceOrientation orientation;
+    if (@available(iOS 13.0, *))
```

Comment:
How about iOS 13.1? Does this logic also match it?

Reply:
It matches iOS 13+ and the asterisk(*) indicated the avaliablility of the declaration on all of the platforms.